canonical/canonical.com
# Add departments overview to /careers

## Done

- Added departments overview to /careers
- Removed redundant "Find your team" section

**Note: Icons to come in future pr when available**
**Must be merged after https://github.com/canonical/canonical.com/pull/701**

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: https://canonical-com-720.demos.haus/careers
- See that the overview matches the design 
- See that links work as expected 

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-534
